server/src/main/resources/application.yml

```diff
@@ -80,6 +76,9 @@ spring:
         order_by.default_null_ordering: last
         temp:
           use_jdbc_metadata_defaults: false
+  web:
+    resources:
+      add-mappings: false
 
```

Comment:
These other changes should be in separate issues. 

Replacing deprecated stuff can be more complicated than just changing this file.

Reply:
Reverted back to how it was before - will create new issue